### PR TITLE
RN 0.60.5

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] desktop
- [x] desktop storybook
- [x] maybe update push
- [x] ios
- [x] android 
- [x] tests (mostly work but emoji-mart update changed the api, will need to fix)
- [x] rn-storybook (kinda busted but it seems to mostly work. babel builds etc. just the web server doesn't seem to update the app) you have to refresh the web page and it works, kinda annoying but fine short term

notes:
Android
- [x] test push
- [x] test push sound
- [x]  fix for @unimodules/react-native-adapter: https://github.com/unimodules/react-native-unimodules/issues/52#issuecomment-503495466 (must run after yarn modules) TODO fork

### CODE REVIEW

File: shared/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,6 +11,11 @@
     <uses-permission android:name="android.permission.READ_CONTACTS" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
+    <meta-data  android:name="com.dieam.reactnativepushnotification.notification_channel_name"
```
Comment (chrisnojima) [inline shared/android/app/src/main/AndroidManifest.xml]: @MarcoPolo maybe not needed after your changes?

**Comment by chrisnojima** [comment]
- [ ] after release

**Comment by chrisnojima** [comment]
- [ ] crashes on device

**Comment by chrisnojima** [comment]
- [ ] text files failed

File: shared/android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (chrisnojima) [reply shared/android/app/src/main/AndroidManifest.xml]: @MarcoPolo bump

Comment (MarcoPolo) [reply shared/android/app/src/main/AndroidManifest.xml]: Oh yeah, kill this

**Pull request merged**